### PR TITLE
Login validation

### PR DESCRIPTION
**Pull request by zfreeds**

### CODE REVIEW

File: security/templates/security/login.html [hunk 1]
```diff
@@ -1,23 +1,34 @@
...
+                    <input name="password" type="password" class="form-control" id="exampleInputPassword1"
+                           placeholder="Password" required>
+                </div>
+                <button type="submit" class="btn btn-primary col-12">Submit</button>
+                <div class="row">
+                    <a class="mx-auto" href="#">Forgot your password?</a>
+                </div>
+            </form>
+            <br>
+            {% if messages %}
```
Comment (justin-gerhardt) [inline security/templates/security/login.html]: The alerts should probably be at the top

Comment (zfreeds) [reply security/templates/security/login.html]: It looks funky and pushes things down, so I put it in the jumbotron. I'm down to change it but maybe take a look at it

Comment (justin-gerhardt) [reply security/templates/security/login.html]: In the jumbotron is fine, I just think it should be above the form. I've pushed a commit have a look

Comment (zfreeds) [reply security/templates/security/login.html]: Looks good, feel free to approve and merge

**Review by justin-gerhardt — APPROVED** [review]

**Pull request merged**